libs/openFrameworks/utils/ofSystemUtils.cpp

```diff
@@ -378,14 +378,14 @@ ofFileDialogResult ofSystemLoadDialog(string windowTitle, bool bFolderSelection,
 		wchar_t szFileName[MAX_PATH];
 		wchar_t szTitle[MAX_PATH];
 		if(defaultPath!=""){
-			wcscpy_s(szFileName,convertNarrowToWide(ofToDataPath(defaultPath)).c_str());
```

Comment:
Did you check if this change also compiles on VS? 
I remember from a while back VS would require one version of a wide string related function and CB would require another.